packages/victory/src/index.ts

```diff
@@ -1,11 +1,17 @@
 export {
   addEvents,
+  Arc,
+  type ArcProps,
```

Comment:
FWIW, I had to use the `type` annotation here, because we have `--isolatedModules` enabled.  Kinda muddies up the water, but it's also kinda nice and explicit.